ajo/content-cards/README.md

```diff
@@ -0,0 +1,228 @@
+# Content Cards
+
+## Overview
+
+This sample demonstrates using Adobe Experience Platform to get Content Cards from Adobe Journey Optimizer (AJO). It uses the [Adobe Experience Platform Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) to get personalization content and to render it entirely client-side.
+
+Here is what the page looks like on first page load.
+
+<img src=".assets/cards1.png" alt="drawing" width="800"/>
+
+If you click the "Deposit Funds" and "Share on social media" buttons, additional content cards will be shown.  This is because those cards utilize client side triggers so that they show only when certain conditions are met.
+
+## Running the sample
+
+<small>Prerequisite: [install node and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).</small>
+
+To run this sample:
+
+1. [Setup local SSL certificates for https](../../LocalSSLCertificateSetup.md).
+2. Clone the repository to your local machine.
+3. Open a terminal and change directory to this sample's folder.
+4. Run `npm install`
+5. Run `npm start`
+6. Open a web browser to [https://localhost](https://localhost)
+
+## How it works
+
+1. [Web SDK](https://experienceleague.adobe.com/docs/experience-platform/edge/home.html) is included and configured on the page. The configuration is based on the `.env` file within the sample folder.
+
+```javascript
+<script src="https://cdn1.adoberesources.net/alloy/2.18.0/alloy.min.js" async></script>
```

Comment:
Should we change the version to use?

Reply:
yes thanks for pointing that out.  We will change it once content cards has been released for web sdk and prior to merging this PR.